spec/support/fakefs_autoload.rb

```diff
@@ -0,0 +1,20 @@
+# Make FakeFS play nicely with autoload
```

Comment:
Is there a source for this file?

Reply:
What do you mean? I wrote it. I have found any existing solution for FakeFS to handle ActiveSupport autoload magic.

Reply:
I thought it might have been copied from StackOverflow or something. If not, then nothing to do :)

Reply:
We can put it on StackOverflow! :D